### PR TITLE
Amount of adults is added to calculation when counting taken or approved seats.

### PR DESCRIPTION
**Pull request by nikomakela**
Pull reguest for Feature/PT-680. Updated Occurrence model seat count functions and tests related to them. It was also needed to update test-snapshots.

### CODE REVIEW

File: occurrences/tests/test_models.py [hunk 1]
```diff
@@ -42,3 +42,26 @@ def test_enrolment_creation(mock_get_event_data):
     EnrolmentFactory()
     assert Occurrence.objects.count() == 1
     assert StudyGroup.objects.count() == 1
+
+
+@pytest.mark.django_db
+def test_occurrence_seat_taken():
+    EnrolmentFactory()
+    occurrence = Occurrence.objects.all()[0]
```
Comment (quyenlq) [inline occurrences/tests/test_models.py:50]: you can get the `enrolment = EnrolmentFactory()`, then `ocurrence = enrolment.occurrence` and `study_group=enrolment.study_group`. That's look cleaner IMO

File: occurrences/tests/test_models.py [hunk 2]
```diff
@@ -42,3 +42,26 @@ def test_enrolment_creation(mock_get_event_data):
...
+    assert occurrence.seats_taken > 0
+    assert (
+        occurrence.seats_taken == study_group.group_size + study_group.amount_of_adult
+    )
+
+
+@pytest.mark.django_db
+def test_occurrence_seat_approved():
+    EnrolmentFactory(status=Enrolment.STATUS_APPROVED)
+    occurrence = Occurrence.objects.all()[0]
```
Comment (quyenlq) [inline occurrences/tests/test_models.py:61]: Similar to [L50](https://github.com/City-of-Helsinki/palvelutarjotin/pull/105/files#diff-7aaed390c4a28b80908979c3cdb2208c8c353977f71781342c59f697d20b7c3dR50)

**Review by quyenlq — COMMENTED** [review]
Looks good, just need to update the `validation_enrolment` then this should be ready to go

File: occurrences/tests/test_models.py (hunk 1 above)
Comment (nikomakela) [reply occurrences/tests/test_models.py:50]: You are absolutely right!

**Review by quyenlq — APPROVED** [review]
Looks great now!

**Pull request merged**